[MTM-62844] Move localization docs to administration

luja-c8y opened the pull request:
I moved this doc: https://github.com/Cumulocity-IoT/c8y-docs/pull/2938/files to admin because a feature was moved there.

https://cumulocity-iot-c8y-docs-build-pr-3017.surge.sh/standard-tenant/changing-settings/#localization

Commit:
Added docs page

Commit:
Remove redundant docs parts

reey commented on content/standard-tenant/changing-settings-bundle/localization.md:
```diff
@@ -0,0 +1,78 @@
+- label: localization
+  title: Localization
+  content: "
+  Use the **Localization** feature to add translations for asset properties or asset models, as well as to add custom translations for the existing static text in the DTM application.
```
I would suggest removing the `DTM application` here..

luja-c8y replied:
done

reey commented on content/standard-tenant/changing-settings-bundle/localization.md:
```diff
@@ -0,0 +1,78 @@
+   To add a new translation, click **Add translation** on the top menu bar. In the resulting dialog box, add the name of the key, followed by the translation in the respective field.
+
+
+   Note that to use this feature, your tenant must have the Public-options application installed."
```
There is no need for a `Public-options application`.

luja-c8y replied:
done

Commit:
Remarks

luja-c8y commented:
> @luja-c8y If you moved the content, shouldn´t it then also be removed from somewhere else?

@BeateRixen it was already removed few weeks ago: https://github.com/Cumulocity-IoT/c8y-docs/pull/2938 from DTM docs

Commit:
Remove redundant info

luja-c8y commented:
> > > @luja-c8y If you moved the content, shouldn´t it then also be removed from somewhere else?
> > 
> > 
> > @BeateRixen it was already removed few weeks ago: #2938 from DTM docs
> 
> @luja-c8y But is the feature still only relevant for DTM? At least the text says so. Then this should be clearly stated at the top.

@BeateRixen I removed info part from a bottom that suggest correlation with DTM. Feature works globally:
![image](https://github.com/user-attachments/assets/0935d792-d8b9-4361-b62f-e7c96f5809e3)
![image](https://github.com/user-attachments/assets/c122e131-674e-4feb-9c19-5ff7caeafe5b)

luja-c8y committed:
Merge branch 'develop' into feat/MTM-62844-move-localization-view-docs-to-administration

reey commented on content/standard-tenant/changing-settings-bundle/localization.md:
```diff
@@ -0,0 +1,70 @@
+- label: localization
+  title: Localization
+  content: "
+  Use the **Localization** feature to add translations for asset properties or asset models, as well as to add custom translations for the existing static text in the application.
+
+
+   Once you created all asset properties and asset models, their names are added under **Identifier** in the **Localization** page. You can add further translations as required.  
```
@luja-c8y can you remove all asset references here? This UI is not specific to assets..

luja-c8y replied:
Done

Commit:
Remove asset related text

Commit:
Merge branch 'feat/MTM-62844-move-localization-view-docs-to-administration' of github.com:SoftwareAG/c8y-docs into feat/MTM-62844-move-localization-view-docs-to-administration

BeateRixen commented on content/standard-tenant/changing-settings-bundle/localization.md:
```diff
@@ -0,0 +1,66 @@
+
+### Localization {#localization}
+
+Click **Localization** in the **Settings** to open localization settings.
```
```suggestion
Click **Localization** in the **Settings** menu in the navigator to open the translation editor.
```

BeateRixen commented on content/standard-tenant/changing-settings-bundle/localization.md:
```diff
@@ -0,0 +1,66 @@
+
+Click **Localization** in the **Settings** to open localization settings.
+
+Use the **Localization** feature if you want to add custom translations for the existing static text in the application.
```
```suggestion
```

BeateRixen commented on content/standard-tenant/changing-settings-bundle/localization.md:
```diff
@@ -0,0 +1,66 @@
+   To add a new translation, click **Add translation** on the top menu bar. In the resulting dialog box, add the name of the key, followed by the translation in the respective field.
+
+---
+
```
```suggestion
Using the **Localization** functionality you can add custom translations for existing static text in the UI.
```

BeateRixen commented on content/standard-tenant/changing-settings-bundle/localization.md:
```diff
@@ -0,0 +1,66 @@
+- label: localization
+  title: Localization
+  content: "
+  Use the **Localization** feature to add translations for the existing static text in the application.
```
```suggestion
  Use the **Localization** feature to add translations for the existing static text in the platform UI.
```

BeateRixen commented on content/standard-tenant/changing-settings-bundle/localization.md:
```diff
@@ -0,0 +1,66 @@
+
+  {{< /c8y-admon-req >}}
+
+### Localization {#localization}
```
```suggestion
```

BeateRixen commented on content/standard-tenant/changing-settings-bundle/localization.md:
```diff
@@ -0,0 +1,66 @@
+  content: "
+  Use the **Localization** feature to add translations for the existing static text in the application.
+
+   To add a new translation, click **Add translation** on the top menu bar. In the resulting dialog box, add the name of the key, followed by the translation in the respective field.
```
```suggestion
   To add a new translation, click **Add translation** on the top menu bar. In the resulting dialog box, add the name of the key, followed by the translations in the respective fields.
```

BeateRixen commented on content/standard-tenant/changing-settings-bundle/localization.md:
```diff
@@ -0,0 +1,66 @@
+
+![Main view localization page](/images/users-guide/Administration/admin-localization-main-page.png)
+
+To add a new translation in the application, you can add the text as key and add the translations as required.
```
```suggestion
```

BeateRixen commented on content/standard-tenant/changing-settings-bundle/localization.md:
```diff
@@ -0,0 +1,66 @@
+
+To add a new translation in the application, you can add the text as key and add the translations as required.
+
+Click **Apply** to apply the changes.
```
```suggestion
```

BeateRixen commented on content/standard-tenant/changing-settings-bundle/localization.md:
```diff
@@ -0,0 +1,66 @@
+
+### To add new identifier for translations {#to-add-new-keys-for-translations}
+
+Add a new identifier and their respective translations using the **Add translation** option on the top right.
```
1. Click **Add translation** on the top menu bar. 
2. In the resulting dialog box, add a name for the new translation key.
3. Optionally, add translations in the respective fields. 
4. Click **Add** to close the the translation editor.
5. Click **Save and apply in the top menu bar to save and the new translation identifier and apply it in the UI.

BeateRixen replied:
```suggestion
1. Click Add translation on the top menu bar.
2. In the resulting dialog box, add a name for the new translation key.
3. Optionally, add translations in the respective fields.
4. Click Add to close the the translation editor.
5. Click **Save and apply in the top menu bar to save and the new translation identifier and apply it in the UI.
```

BeateRixen commented on content/standard-tenant/changing-settings-bundle/localization.md:
```diff
@@ -0,0 +1,66 @@
+
+Add a new identifier and their respective translations using the **Add translation** option on the top right.
+
+Fill all the mandatory fields in the **Add translations** dialog and click **Save**. The new key is saved including the added translations.
```
```suggestion
```

BeateRixen commented on content/standard-tenant/changing-settings-bundle/localization.md:
```diff
@@ -0,0 +1,66 @@
+
+Fill all the mandatory fields in the **Add translations** dialog and click **Save**. The new key is saved including the added translations.
+
+Next, click **Apply** to apply the changes.
```
```suggestion
```

BeateRixen commented on content/standard-tenant/changing-settings-bundle/localization.md:
```diff
@@ -0,0 +1,66 @@
+2. Add or edit the translation for any language.
+3. Click **Save**.
+
+Click **Apply** to apply the changes.
```
```suggestion
Click **Save & apply** to apply the changes.
```

BeateRixen commented on content/standard-tenant/changing-settings-bundle/localization.md:
```diff
@@ -0,0 +1,66 @@
+
+You can add or edit translations for every identifier in two ways:
+
+1. Hover over the respective column for the edit icon <i class="dlt-c8y-icon-edit1 text-primary icon-20"></i> to be visible.
```
```suggestion
1. Hover over the respective column to display the edit icon <i class="dlt-c8y-icon-edit1 text-primary icon-20"></i>.
```

BeateRixen commented on content/standard-tenant/changing-settings-bundle/localization.md:
```diff
@@ -0,0 +1,66 @@
+2. Click the icon to select the desired language.
+3. Add or edit the translation.
+4. Click **Save**.
+
```
```suggestion
5. Click **Save & apply** to apply the changes.
```

BeateRixen commented on content/standard-tenant/changing-settings-bundle/localization.md:
```diff
@@ -0,0 +1,66 @@
+
+or:
+
+1. Click the edit icon <i class="dlt-c8y-icon-edit1 text-primary icon-20"></i> in each row to open the **Edit translations** dialog window.
```
```suggestion
1. Click the edit icon <i class="dlt-c8y-icon-edit1 text-primary icon-20"></i> in any row to open the translation editor.
```

BeateRixen commented on content/standard-tenant/changing-settings-bundle/localization.md:
```diff
@@ -0,0 +1,66 @@
+1. Click the edit icon <i class="dlt-c8y-icon-edit1 text-primary icon-20"></i> in each row to open the **Edit translations** dialog window.
+2. Add or edit the translation for any language.
+3. Click **Save**.
+
```
```suggestion
4. Click **Save & apply** to apply the changes.
```

BeateRixen commented on content/standard-tenant/changing-settings-bundle/localization.md:
```diff
@@ -0,0 +1,66 @@
+
+Click **Apply** to apply the changes.
+
+Once all your translations are done, you can change the language under **UI settings** to see the translations in action. For more information see [To change user settings](/get-familiar-with-the-ui/user-settings/#to-change-user-settings).
```
```suggestion
To view the added or modified translations in the UI, change the language from the user menu, see [To change user settings](/get-familiar-with-the-ui/user-settings/#to-change-user-settings).
```

BeateRixen committed:
Apply suggestions from code review

BeateRixen committed:
Update localization.md

BeateRixen committed:
Update localization.md

BeateRixen commented on content/standard-tenant/changing-settings-bundle/localization.md:
```diff
@@ -0,0 +1,60 @@
+
+1. Click the edit icon <i class="dlt-c8y-icon-edit1 text-primary icon-20"></i> in any row to open the translation editor for the respective identifier.
+2. Add or edit the translations.
+3. Click **Add** to close the the translation editor.
```
```suggestion
3. Click **Add** to close the translation editor.
```

BeateRixen committed:
Update content/standard-tenant/changing-settings-bundle/localization.md

BeateRixen commented:
I have rewritten most of the text because it did not work as described in various aspects. 

There is one open point left: in the first screenshot, the orange border seems a bit random as you do not necessarily have to fill in these 3 fields. I would actually remove it completely. Did you create it, @luja-c8y, and could you please update it?

Commit:
Remove ss and fixed docs build

luja-c8y commented:
> I have rewritten most of the text because it did not work as described in various aspects.
> 
> There is one open point left: in the first screenshot, the orange border seems a bit random as you do not necessarily have to fill in these 3 fields. I would actually remove it completely. Did you create it, @luja-c8y, and could you please update it?

Thank you @BeateRixen . Yes, I recreated it based on original screenshots, and I deleted it now.

Commit:
Bringed back and updated screenshots

Commit:
Merge branch 'feat/MTM-62844-move-localization-view-docs-to-administration' of github.com:SoftwareAG/c8y-docs into feat/MTM-62844-move-localization-view-docs-to-administration

Commit:
Updated screenshots
